WMFData/Sources/WMFData/Models/Donor Experience/WMFDonateConfig.swift

```diff
@@ -12,4 +12,16 @@ public struct WMFDonateConfig: Codable {
     public func transactionFee(for currencyCode: String) -> Decimal? {
         return currencyTransactionFees[currencyCode] ?? currencyTransactionFees["default"]
     }
+
+    public func getMaxAmount(for currencyCode: String) -> Decimal {
+        var max = currencyMaximumDonation[currencyCode] ?? Decimal()
+
+        if max.isZero {
+            if let defaultMin = currencyMinimumDonation["USD"] {
+                max = (currencyMinimumDonation[currencyCode] ?? 0.0) / defaultMin *
+                (currencyMaximumDonation["USD"] ?? 0.0)
```

Comment:
Out of curiosity, does defaulting to 0.0 cause any issues? I would worry about it trying to divide by zero and causing an error/crashing.

Reply:
Good catch, updated